README.md

```diff
@@ -1,14 +1,29 @@
 # artsy-wwwify
 A tiny app that 301 redirects requests to www.artsy.net
 
-### Deploy
-
-This app is deployed on Amazon OpsWorks. [Click here to deploy](https://console.aws.amazon.com/opsworks/home?region=us-east-1#/stack/7db728d1-44a8-49e0-8362-92e4723c898e/deployments/deploy-app).
+### Meta
+* State: production
+* Production: https://artsy.net
+* Staging: N/A
```

Comment:
Why is staging N/A? Looks like line #9 is referring to a staging environment, is it only a branch?

Reply:
This app exists to respond to requests for the `artsy.net` domain without any subdomain specified. A typical staging app will live at a `...-staging.artsy.net` subdomain so it's not clear how a staging version of this app would be useful or testable (unless, e.g., we hosted it at a completely different "naked" domain, like artsystaging.net). The staging deployment will get a load-balancer automatically, but I'm just not planning to configure a special subdomain for it.